README.Rmd

```diff
@@ -26,45 +26,15 @@ knitr::read_chunk(path = "inst/plumber/capitalize/plumber.R",
 [![Codecov test coverage](https://codecov.io/gh/rstudio/plumbertableau/branch/main/graph/badge.svg)](https://codecov.io/gh/rstudio/plumbertableau?branch=main)
 <!-- badges: end -->
 
-An R package that helps create [Plumber APIs](https://www.rplumber.io/) that are
-compliant with the [Tableau Analytics Extension
-API](https://tableau.github.io/analytics-extensions-api/#).
-
-## Installation
-
-You can install the development version of plumbertableau from
-[GitHub](https://github.com/rstudio/plumbertableau) with:
-
-
-```{r}
-remotes::install_github("rstudio/plumbertableau")
-```
-
```

Comment:
I think we'll still want the Installation instructions here - that's pretty common practice for R package README pages

Reply:
Sounds good. I'll add those back in.